models/public/background-matting-mobilenetv2/model.yml

```
@@ -0,0 +1,36 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  The "background-matting-mobilenetv2" model is a high-resolution background replacement technique based on
+  background matting (with MobileNetV2 backbone), where an additional frame of the background is
+  captured and used in recovering the alpha matte and the foreground layer. This model is
+  pre-trained in PyTorch* framework and converted to ONNX* format. More details provided in
+  the paper <https://arxiv.org/abs/2012.07810>.
+  For details see the repository <https://github.com/DmitriySidnev/BackgroundMattingV2>.
+task_type: background_matting
+files:
+  - name: bgm_mobilenetv2.onnx
+    size: 20006715
+    checksum: 3fc7bad659149e3e9e0feb4ef531c177ab158e5bbcfdabba9952c3832053f353d23560ff488aeae857245734ef639e6b
+    source: https://github.com/DmitriySidnev/BackgroundMattingV2/raw/master/onnx/bgm_mobilenetv2.onnx
+model_optimizer_args:
+  - --input_shape=[1,3,720,1280],[1,3,720,1280]
+  - --input=src,bgr
```

now we also provide --layout parameter for mo arguments
```suggestion
  - --input=src,bgr
  - --layout=src(NCHW),bgr(NCHW)
```